client/src/js/jobs/components/Item/Item.js

```diff
@@ -25,7 +25,9 @@ const JobItemHeader = styled.div`
 `;
 
 const JobItemLinkBox = styled(LinkBox)`
-    padding: 0;
+    padding-top: 5px;
+    padding-left: 0;
+    padding-bottom: 0;
```

Comment:
These should be reduce to a single line (https://developer.mozilla.org/en-US/docs/Web/CSS/padding).